go.mod

```diff
@@ -7,8 +7,10 @@ require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/gin-gonic/gin v1.6.3
-	github.com/go-vela/mock v0.7.4
+	github.com/go-vela/mock v0.7.5-0.20210325145820-d14ee5b79d42
+	github.com/go-vela/server v0.7.5-0.20210323124812-0da2c57a87ff
```

Comment:
@JordanSussman with regards to adding server as a dependency, could we consider moving the `database.BuildQueue` into `go-vela/types`?
from: https://github.com/go-vela/server/blob/master/database/build.go#L433-L438

I bring it up because vendoring `server` into `sdk-go` will impact our release/tagging workflow and cause some trickle down changes.
Which I'll admit isnt a huge deal but something that was brought up in team discussion.